integration_tests/tests/consistency_twilio__number_overview.sql

```diff
@@ -0,0 +1,52 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+{% set exclude_cols = var('consistency_test_exclude_metrics', []) %}
```

Comment:
Suggestion to hard code.
```suggestion
{% set exclude_cols = ['total_spend'] + var('consistency_test_exclude_metrics', []) %}
```